codejam/client/widgets/whiteboard_screen.py

```diff
@@ -113,6 +118,11 @@ def on_enter(self) -> None:
     def on_pre_leave(self) -> None:
         """Called when the screen is about to be hidden."""
         Window.unbind(mouse_pos=self.mouse_pos)
+        if self.current_trick:
```

Comment:
you were supposed to call a function not do it like this, trick can be also in clock and pacman animation